.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    name: Build and Push Docker Images
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Setup Gradle Cache
+      uses: actions/cache@v1
+      with:
+        path: ~/.gradle/caches
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+        restore-keys: |
+          ${{ runner.os }}-gradle-
+    - name: Log in to Docker Hub
+      run: docker login -u '${{ secrets.REGISTRY_USER }}' -p '${{ secrets.REGISTRY_PASS }}' '${{ secrets.REGISTRY_URL }}'
```

Comment:
I had to add a snippet to enable buildkit here:
```patch
diff --git a/.github/workflows/release.yml b/.github/workflows/release.yml
index d2b81ce..3cc551f 100644
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -23,6 +23,10 @@ jobs:
           ${{ runner.os }}-gradle-
     - name: Log in to Docker Hub
       run: docker login -u '${{ secrets.REGISTRY_USER }}' -p '${{ secrets.REGISTRY_PASS }}' '${{ secrets.REGISTRY_URL }}'
+    - name: Enable buildkit
+      shell: bash
+      run: |
+        echo '{"experimental": "enabled"}' > ~/.docker/config.json
     - name: Set TAG Environment Variable
       run: |
         export TAG=$(echo "${GITHUB_REF}" | cut -d "/" -f3)

```

Reply:
Shouldn't be necessary, but definitely doesn't hurt as an environment is set https://github.com/Islandora-Devops/isle-gradle-docker-plugin/blob/main/src/main/kotlin/ca.islandora.gradle.docker.gradle.kts#L293 to force the use of buildkit. More likely it was GPG key servers failing to respond causing a failed build as this is often a problem for me. There is a ticket for the GPG issue: https://github.com/Islandora-Devops/isle-buildkit/issues/98

Reply:
As evidence our current workflow file doesn't do this and still passes: https://github.com/Islandora-Devops/isle-buildkit/actions/runs/409228695/workflow